openpype/lib/attribute_definitions.py

```diff
@@ -443,7 +445,10 @@ def __init__(self, key, items, default=None, **kwargs):
 
         items = self.prepare_enum_items(items)
         item_values = [item["value"] for item in items]
-        if default not in item_values:
+        if multiselection and default is None:
+            default = []
+
+        if not multiselection and default not in item_values:
             for value in item_values:
                 default = value
                 break
```

Comment:
I'm pretty sure looking at this logic that there's currently no `default value` check for `defaults` if `multiselection` were enabled.
```suggestion
        if multiselection:
            if not isinstance(default, (list, tuple, set)):
                # allow single value as default
                default = [default]
            default = list(item_values.intersection(default))
        
        elif default not in item_values:
            # single selection; use first value if no valid default
            default = next(iter(item_values), None)
```
Note that this pseudocode logic assumes `item_values` is a `set` already

Reply:
If you pass `multiselection=True`, you should be able pass `default=[value]`...